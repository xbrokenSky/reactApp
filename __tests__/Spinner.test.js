import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../src/scripts/components/spinner';

describe('Check Spinner UI-component', () => {
    it('Correct renders component', () => {
        const component = shallow(<Spinner />);
        expect(component).toMatchSnapshot();
    });
});
