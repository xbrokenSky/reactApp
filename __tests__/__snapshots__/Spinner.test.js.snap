// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check Spinner UI-component Correct renders component 1`] = `
<div
  className="spinner"
>
  <div
    className="lds-css ng-scope"
  >
    <div
      className="lds-ripple"
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    >
      <div />
      <div />
    </div>
  </div>
</div>
`;
